services/appmetrics/api.go

```diff
@@ -3,16 +3,19 @@ package appmetrics
 import (
 	"context"
 
+	"github.com/pborman/uuid"
```

Comment:
This repo uses the Google UUID package. Does this package have any special functionality missing from the Google package?

Reply:
There is nothing special, we only need a UUID. I just discovered a usage in `account/accounts.go` and replicated it.

Reply:
Ah, that's annoying. So we've already got multiple dependencies that server the same purpose 😬, ok.

Flawless PR, then. 😆